README.md

```diff
@@ -251,22 +251,13 @@ tox
 
 #### Linting
 
-To fix imports using ruff:
+To format code and sort imports, use:
 
 ```sh
 pip install -r requirements.txt
-ruff check --select="I" --fix .
+ruff format .
```

Comment:
I believe you needs two sections. This is the "Linting" section. Which should explain .. well the linter :) https://docs.astral.sh/ruff/linter/

So:

```suggestion
ruff check --fix 
```

Reply:
And next, there should be a Second  new section (`#### Formatter`) about the formatter: https://docs.astral.sh/ruff/formatter/

```sh
ruff format
```

Right?